pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -26,21 +26,21 @@ cdef kh{{name}}_t to_kh{{name}}_t({{name}}_t val) nogil:
 
 
 # name
-c_types = ['khcomplex128_t',
```

Comment:
i'd rather have just one variable here rather than a tuple.  with a tuple i have to check if they all follow the apparent pattern or if e.g. bool/object are special-cases

Reply:
good call, thanks, have reverted (and it's not too bad to apply the substitutions to the included template files and then parse them, so I've gone with that)